pipelines/internal/commands/run/run.go

```diff
@@ -357,6 +358,10 @@ func buildRequest(filename, project string) (*genomics.RunPipelineRequest, error
 		Labels:         vmLabels,
 	}
 
+	if channel := *cosChannel; channel != "" {
+		vm.BootImage = "projects/cos-cloud/global/images/family/cos-" + channel
```

Comment:
@kemp-google I'm not sure it's really a channel, I think it's more of an image family, but for now it's okay.